verification/curator-service/ui/src/components/Case.tsx

```diff
@@ -95,7 +100,7 @@ export interface Case {
     sources: Source[];
     travelHistory: TravelHistory;
     genomeSequences: GenomeSequence[];
+    pathogens: Pathogen[];
```

Comment:
pathogens?: Pathogen[];

Reply:
We haven't used ? for any other array fields like sources or genomeSequences. Any reason for it here in particular?

Reply:
we should have been using those, if a case has no pathogen it won't be returned by the server os we should mark it as such to make sure that everywhere we access it we do check that it is present, this is what you're doing in this PR as well, you're using ? to access the field even though you didn't add it here it's inconsistent so let's add ? here.

Reply:
We use this Case object for EditCase and we're able to load that page with a partial case without a problem, but happy to add for general defensive code. Done.

Reply:
After a couple attempts I actually don't think it's a good idea to do this. It requires adding ? or undefined in loads of places. We don't have any issues with the server returning partial cases right now, so I don't think this helps us.

Reply:
Well yes if you say this can be undefined you have to check for undefinedess everywhere that's WAI. If you say the server ALWAYS returns the pathogens field even when empty then we're fine and we don't need "?", otherwise we'll see stuff fail down the road.

Reply:
I just checked loading the EditCase page with a partial case (and found a couple missing ? along the way which I'll commit). Interestingly, when I printed the Case returned from the server it didn't have a pathogens field, however there were no errors either. Perhaps an object can always have a field not set, and that's different from setting an undefined value.